src/Controls/src/Core/Element/Element.cs

```diff
@@ -266,20 +277,22 @@ internal Element ParentOverride
 			}
 		}
 
-		// you're not my real dad
-		/// <include file="../../docs/Microsoft.Maui.Controls/Element.xml" path="//Member[@MemberName='RealParent']/Docs/*" />
+		/// <summary>For internal use by the Microsoft.Maui.Controls platform.</summary>
```

Comment:
Can we describe what this actually does and add this as a remark?

Its a public API, people should be able to use it 🤷‍♂️ 

Same for a couple of others below

Reply:
I spoke about this with @PureWeen. They should be made internal, but that will cause compile issues. I wanted to keep the scope of the PR to only docs.

Reply:
Maybe we can mark all these internal things as obsolete too?

Reply:
`[EditorBrowsable(EditorBrowsableState.Never)]` means the API won't show up in intellisense. However, it will show up in the API docs. Not sure it's worth doc'ing something that only appears in one of the two information sources.